system/src/system_cloud_internal.cpp

```diff
@@ -1267,22 +1284,7 @@ int Spark_Handshake(bool presence_announce)
         }
```

Comment:
No idea if these are still relevant events to publish, as that seems to be the difference between resuming session and not.

Reply:
Publishing spark/device/ident is 100% dead code. It's from the 2015 PR https://github.com/particle-iot/device-os/pull/621

The device service doesn't do anything with this event if it was ever published which it won't since `HAL_Get_Device_Identifier` is an empty function on all platforms.

Reply:
spark/device/claim/code is still a live code path but we don't use claim codes in the latest dev tools. We shouldn't delete this code, but it would be ok to send the claim code whether the connection is resumed or new. The device service will respond with spark/device/claim/confirm when it receives the claim code. When the device receives this, it will clear the claim code using `HAL_Set_Claim_Code(NULL)` and mark the device as claimed.